app/templates/partials/applications_table.html

```diff
@@ -27,7 +27,7 @@
 		{% endset %}
 
 		{% if not application.status == 'SUBMITTED' and not is_past_submission_deadline %}
-			{% set application_link = '<a class="govuk-link" href="{}">{}</a>'.format(url_for('application_routes.tasklist', application_id=application.id), gettext('Continue application')) %}
+			{% set application_link = '<a class="govuk-link" href="{}">{}</a>'.format(url_for('application_routes.tasklist', application_id=application.id, fund=fund_short_name, round=round_short_name), gettext('Continue application')) %}
```

Comment:
why do we pass fund and round here?

Reply:
so we can re-direct user back to the fund round page (/accont?fund=blah&round=blah) as opposed to the all applications page (/account) + we want to hide access to this page until we run concurrent live funds 👍🏾

Reply:
ah okay so the logic doesnt exist on tasklist.html to pick this up yet?